### ISSUE TITLE
Reduce redundant steps

### ISSUE THREAD

**Issue by spyoungtech** [spyoungtech/behave-webdriver#16]
Some steps contain exactly the same logic. Instead of repeating the logic, this should be refactored. For example 
```
@then('I expect that element "([^"]*)?"( not)* matches the text "([^"]*)?"')
```

Contains the same logic as

```
@given('the element "([^"]*)?"( not)* matches the text "([^"]*)?"')
```

It would be ideal if instead, we did something like

```
@then('I expect that element "([^"]*)?"( not)* matches the text "([^"]*)?"')
@given('the element "([^"]*)?"( not)* matches the text "([^"]*)?"')
def check_element_matches_text(context, element, negative, text):
    # logic
```

These are the 6 redundancies identified by Better Code Hub. They are all between `given.py` and `then.py`

- [x] 12 lines of code: ([given](https://github.com/spyoungtech/behave-webdriver/blob/7504e4069939fd22184a863da33c4d9fe52b640c/behave_webdriver/steps/given.py#L91-L102), [then](https://github.com/spyoungtech/behave-webdriver/blob/7504e4069939fd22184a863da33c4d9fe52b640c/behave_webdriver/steps/then.py#L56-L67)). 
- [x] 10 lines of code: ([given](https://github.com/spyoungtech/behave-webdriver/blob/7504e4069939fd22184a863da33c4d9fe52b640c/behave_webdriver/steps/given.py#L190-L199), [then](https://github.com/spyoungtech/behave-webdriver/blob/7504e4069939fd22184a863da33c4d9fe52b640c/behave_webdriver/steps/then.py#L198-L207)). 
- [x] 7 lines of code: ([given](https://github.com/spyoungtech/behave-webdriver/blob/7504e4069939fd22184a863da33c4d9fe52b640c/behave_webdriver/steps/given.py#L107-L113), [then](https://github.com/spyoungtech/behave-webdriver/blob/7504e4069939fd22184a863da33c4d9fe52b640c/behave_webdriver/steps/then.py#L72-L78)). 
- [x] 7 lines of code: ([given](https://github.com/spyoungtech/behave-webdriver/blob/7504e4069939fd22184a863da33c4d9fe52b640c/behave_webdriver/steps/given.py#L155-L162), [then](https://github.com/spyoungtech/behave-webdriver/blob/7504e4069939fd22184a863da33c4d9fe52b640c/behave_webdriver/steps/then.py#L139-L146)). 
- [x] 6 lines of code: ([given](https://github.com/spyoungtech/behave-webdriver/blob/7504e4069939fd22184a863da33c4d9fe52b640c/behave_webdriver/steps/given.py#L203-L208), [then](https://github.com/spyoungtech/behave-webdriver/blob/7504e4069939fd22184a863da33c4d9fe52b640c/behave_webdriver/steps/then.py#L211-L216)). 
- [x] 6 lines of code: ([given](https://github.com/spyoungtech/behave-webdriver/blob/7504e4069939fd22184a863da33c4d9fe52b640c/behave_webdriver/steps/given.py#L168-L173), [then](https://github.com/spyoungtech/behave-webdriver/blob/7504e4069939fd22184a863da33c4d9fe52b640c/behave_webdriver/steps/then.py#L180-L185)).

**Comment by spyoungtech** [comment]
This will likely require reorganizing steps out of `given.py`, `then.py` and `when.py` into some other organization.

**Comment by spyoungtech** [comment]
These are the 6 redundancies identified by Better Code Hub. They are all between `given.py` and `then.py`

- [ ] 12 lines of code: ([given](https://github.com/spyoungtech/behave-webdriver/blob/7504e4069939fd22184a863da33c4d9fe52b640c/behave_webdriver/steps/given.py#L91-L102), [then](https://github.com/spyoungtech/behave-webdriver/blob/7504e4069939fd22184a863da33c4d9fe52b640c/behave_webdriver/steps/then.py#L56-L67)). 
- [ ] 10 lines of code: ([given](https://github.com/spyoungtech/behave-webdriver/blob/7504e4069939fd22184a863da33c4d9fe52b640c/behave_webdriver/steps/given.py#L190-L199), [then](https://github.com/spyoungtech/behave-webdriver/blob/7504e4069939fd22184a863da33c4d9fe52b640c/behave_webdriver/steps/then.py#L198-L207)). 
- [ ] 7 lines of code: ([given](https://github.com/spyoungtech/behave-webdriver/blob/7504e4069939fd22184a863da33c4d9fe52b640c/behave_webdriver/steps/given.py#L107-L113), [then](https://github.com/spyoungtech/behave-webdriver/blob/7504e4069939fd22184a863da33c4d9fe52b640c/behave_webdriver/steps/then.py#L72-L78)). 
- [ ] 7 lines of code: ([given](https://github.com/spyoungtech/behave-webdriver/blob/7504e4069939fd22184a863da33c4d9fe52b640c/behave_webdriver/steps/given.py#L155-L162), [then](https://github.com/spyoungtech/behave-webdriver/blob/7504e4069939fd22184a863da33c4d9fe52b640c/behave_webdriver/steps/then.py#L139-L146)). 
- [ ] 6 lines of code: ([given](https://github.com/spyoungtech/behave-webdriver/blob/7504e4069939fd22184a863da33c4d9fe52b640c/behave_webdriver/steps/given.py#L203-L208), [then](https://github.com/spyoungtech/behave-webdriver/blob/7504e4069939fd22184a863da33c4d9fe52b640c/behave_webdriver/steps/then.py#L211-L216)). 
- [ ] 6 lines of code: ([given](https://github.com/spyoungtech/behave-webdriver/blob/7504e4069939fd22184a863da33c4d9fe52b640c/behave_webdriver/steps/given.py#L168-L173), [then](https://github.com/spyoungtech/behave-webdriver/blob/7504e4069939fd22184a863da33c4d9fe52b640c/behave_webdriver/steps/then.py#L180-L185)).

**Comment by spyoungtech** [comment]
Fixed in #24 🎉
